example-deck.gl.md

```diff
@@ -0,0 +1,400 @@
+We request all measured carbon monoxide values from Germany from the fourth and fifth of February (this time span is arbitrary chosen) using the [geo.json](https://demo.emissions-api.org/ui/#/default/emissionsapi.web.get_data) API endpoint and the url query parameters *begin*, *end* and *polygon*:
+
+```
+https://demo.emissions-api.org/api/v1/geo.json
+    ?begin=2019-02-04
+    &end=2019-02-06
+    &polygon=9.921906365609232,54.983104153048025,9.9395797054529,54.596641954153256,…,9.921906365609232,54.983104153048025
+```
+
+We are using the *polygon* url query parameter to receive carbon monoxide values from within the border of Germany.
+Further, note that the end date is excluded in the query.
+
+The response should look like this:
+
+```json
+{
+  "features": [
+    {
+      "geometry": {
+        "coordinates": [
+          14.129653,
+          53.374462
+        ],
+        "type": "Point"
+      },
+      "properties": {
+        "carbonmonoxide": 0.0349522158503532,
+        "timestamp": "2019-02-05T11:21:47.923000Z"
+      },
+      "type": "Feature"
+    },
+    {
+      "geometry": {
+        "coordinates": [
+          13.63431,
+          53.800873
+        ],
+        "type": "Point"
+      },
+      "properties": {
+        "carbonmonoxide": 0.0341427326202393,
+        "timestamp": "2019-02-05T11:21:56.562000Z"
+      },
+      "type": "Feature"
+    },
+    …
+  ],
+  "type": "FeatureCollection"
+```
+
+What we get here is a standardized [GeoJSON FeatureCollection](https://geojson.org/).
+GeoJSON objects are often used for handling geographic data structures and are supported by a variety of different tools.
+
+## Requesting Data
+
+After we now know how the data is structured and where we can find it, we have to think about how we get it.
+Luckily enough, the deck.gl object we are using later is able to use the URL and handle the request itself.
+So we only have to construct the URL:
+
+```javascript
+// A polygon describing the vague form of Germany
+const GERMANY = 'polygon=9.921906365609232,54.983104153048025,' +
+    '9.9395797054529,54.596641954153256,' +
+    '10.950112338920519,54.363607082733154,' +
+    …
+    '8.526229282270208,54.96274363872516,' +
+    '9.282048780971136,54.83086538351631,' +
+    '9.921906365609232,54.983104153048025';
+
+// API URL
+const API_URL = 'https://demo.emissions-api.org/api/v1/geo.json?' +
+    GERMANY + '&begin=2019-02-04&end=2019-02-06';
+```
+
+## Plotting Data
+
+We are using the [Mapbox](https://www.mapbox.com/) support, deck.gl offers, and use Mapbox as the base map for our plot.
+Therefore we have to include the Mapbox script and stylesheet:
+
+```html
+<script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.js'></script>
+<link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css' rel='stylesheet' />
+```
+
+and generate a [Mapbox Token](https://docs.mapbox.com/help/how-mapbox-works/access-tokens/) for later use.
```

Comment:
Maybe just mention the token creation when you actually use it.

Reply:
Here mapbox is included, so I think this it is a good place to talk about this token. Otherwise you talk about specific JavaScript code later and then say: *Ah by the way, you have to go to this web service, make a account and create a token*.